src/mojs.babel.js

```diff
@@ -0,0 +1,51 @@
+import h from './h';
+import shapesMap from 'shapes/shapesMap';
+import stagger from './stagger';
+import tweener from 'tween/tweener';
+import easing from 'easing/easing';
+import Shape from './shape';
+import ShapeSwirl from './shape-swirl';
+import Burst from './burst';
+import Html from './html';
+import Spriter from './spriter';
+import MotionPath from './motion-path';
+import Tween from 'tween/tween';
+import Timeline from 'tween/timeline';
+import Tweenable from 'tween/tweenable';
+import Thenable from './thenable';
+import Tunable from './tunable';
+import Delta from 'delta/delta';
+import Deltas from 'delta/deltas';
+import Module from './module';
+
+export default {
+  revision: build.revision,
+  isDebug: true,
+  helpers: h,
+  Shape,
+  ShapeSwirl,
+  Burst,
+  Html,
+  stagger,
+  Spriter,
+  MotionPath,
+  Tween,
+  Timeline,
+  Tweenable,
+  Thenable,
+  Tunable,
+  Module,
+  tweener,
+  easing,
+  shapesMap,
+  _pool: {
+    Delta,
+    Deltas,
+  },
+  h: h,
```

Comment:
can simplify with just `h,` to keep it consistent

Reply:
Sure!